x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiBasicTable,
+  EuiText,
+  EuiTableFieldDataColumnType,
+  EuiToolTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  FormattedMessage,
+  FormattedDate,
+  FormattedTime,
+  FormattedNumber,
+  FormattedRelative,
+} from '@kbn/i18n/react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { usePageId } from '../use_page_id';
+import {
+  selectIsLoading,
+  selectPageIndex,
+  selectPageSize,
+  selectPolicyItems,
+  selectTotal,
+} from '../../store/policy_list/selectors';
+import { usePolicyListSelector } from './policy_hooks';
+import { PolicyListAction } from '../../store/policy_list';
+import { PolicyData } from '../../types';
+import { TruncateText } from '../../components/truncate_text';
+
+interface TTableChangeCallbackArguments {
+  page: { index: number; size: number };
+}
+
+const TruncateTooltipText = styled(TruncateText)`
+  .euiToolTipAnchor {
+    display: block;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+`;
+
+const FormattedDateAndTime: React.FC<{ date: Date }> = ({ date }) => {
+  // If date is greater than or equal to 24h (ago), then show it as a date
+  // else, show it as relative to "now"
+  return Date.now() - date.getTime() >= 8.64e7 ? (
+    <>
+      <FormattedDate value={date} year="numeric" month="short" day="2-digit" />
+      {' @'}
```

Comment:
I think the formatting should be left up to i18n entirely in this case. Consider the 'ar-EG' locale for example. When using date and time together, time comes first in this locale.

![image](https://user-images.githubusercontent.com/35559/74450752-9b683400-4e4c-11ea-8bb2-2edfeb5913cd.png)

In react you could get this treatment with:


```tsx
const FormattedDateAndTime: React.FC<{ date: Date }> = ({ date }) => {
  // If date is greater than or equal to 24h (ago), then show it as a date
  // else, show it as relative to "now"
  return Date.now() - date.getTime() >= 8.64e7 ? (
    <FormattedDate
      value={date}
      year="numeric"
      month="short"
      day="2-digit"
      hour="numeric"
      minute="numeric"
    />
  ) : (
    <FormattedRelative value={date} />
  );
};

```

Reply:
Yeah, that's what I was hoping for, but the UX mocks show that we should display the dates as:
![image](https://user-images.githubusercontent.com/56442535/74462206-d7f05b80-4e5d-11ea-82da-7d147e70bd3c.png)

Do you know if there is an issue if we use the implemented format?

During a UX review, there was also some questions around how the Kibana timezone and date/time format setting might impact display of these.

/cc @bfishel